core/src/main/java/com/linecorp/armeria/client/RefreshingAddressResolverGroup.java

```diff
@@ -99,11 +102,13 @@
     private final Backoff refreshBackoff;
     private final List<DnsRecordType> dnsRecordTypes;
     private final Consumer<DnsNameResolverBuilder> resolverConfigurator;
+    private final PrometheusMeterRegistry metricRegistry;
 
     RefreshingAddressResolverGroup(Consumer<DnsNameResolverBuilder> resolverConfigurator,
                                    int minTtl, int maxTtl, int negativeTtl, long queryTimeoutMillis,
                                    Backoff refreshBackoff,
-                                   @Nullable ResolvedAddressTypes resolvedAddressTypes) {
+                                   @Nullable ResolvedAddressTypes resolvedAddressTypes,
+                                   @Nonnull PrometheusMeterRegistry metricRegistry) {
```

Comment:
- `@Nonnull` is redundant. Everything in our code is nonnull.
- Should accept `MeterRegistry`
- `metricRegistry` -> `meterRegistry`